src/test/java/io/vertx/ext/apex/handler/StaticHandlerTest.java

```diff
@@ -133,7 +133,9 @@ public void testContentHeadersSet() throws Exception {
       String contentType = res.headers().get("content-type");
       String contentLength = res.headers().get("content-length");
       assertEquals("application/json", contentType);
-      assertEquals(18, Integer.valueOf(contentLength).intValue());
+      // Account for CRLF if checked out on Windows
```

Comment:
It's my initial reaction that the files should be identical on all operating systems that they are checked out on otherwise we end up adding operating system specific checks everywhere like this one, which is horrible imho.

I believe pretty much all editors these days including IntelliJ can be configured to preserve line endings so this shouldn't be a problem when editing files.